### DISCUSSION TITLE
How do I import WhereFilterOp, OrderByDirection types in v10?

### DISCUSSION THREAD

**Discussion by tettoffensive** [firebase/firebase-admin-node#1472] [Q&A] [answered]
I'm using the types `WhereFilterOp` and `OrderByDirection` in my project. As I'm upgrading to v10 and now importing this way:

```ts
import { getFirestore, FieldPath } from 'firebase-admin/firestore';
```

I'm running into the issue `Module '"firebase-admin/firestore"' has no exported member 'WhereFilterOp'`, despite being able to see this line `export type WhereFilterOp =` in firestore.d.ts.

What am I missing?

**Comment by lahirumaramba** [comment, marked as answer]
Hey @tettoffensive ,
Thank you for pointing this out. The types need to be exposed from the `firestore` module. Should be fixed in #1513
